controller/user.js

```diff
@@ -16,19 +18,20 @@ export const createUser = async (req, res) => {
 
     const oneEmailOnly = await Users.findOne({ where: { email } });
 
-    if (!oneEmailOnly && password === reEnterPassword) {
+    if (oneEmailOnly) return res.status(500).json({ message: 'Email already registered' });
+
+    if (password === reEnterPassword) {
```

Comment:
this validation along with other validations should be done in a `middleware` outside the controller.